docker-compose.yml

```diff
@@ -24,17 +24,12 @@ services:
       - ./docker/rskj/node.conf:/etc/rsk/node.conf
       - ./docker/rskj/data:/var/lib/rsk/.rsk
   
-  dev-liquidity-token-watcher:
-    image: "matterlabs/dev-liquidity-token-watcher:latest"
+  dev-ticker:
+    build:
+      context: .
+      dockerfile: ./docker/dev-ticker/Dockerfile
```

Comment:
Im not sure if we need to build the image each time.

Reply:
@franciscotobar this will be changed in your PR, once we merge it.